### DISCUSSION TITLE
Feed to show updates on packages that I have used in personal and organizational project (this is optional)

### DISCUSSION THREAD

**Discussion by skngetich** [community/community#13155] [Other Feature Feedback, Questions, & Ideas] [answered]
As developer we get updates on security updates. It would be nice to have feed that shows updates such as updated version of the packages for packages that used in the project

**Comment by jfullstackdev** [comment, marked as answer]
not very sure of that but

there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

**Reply by skngetich** [reply]
I did not notice it was there. Thank you
